.pre-commit-config.yaml

```
@@ -21,28 +21,28 @@ repos:
 
 - repo: https://github.com/astral-sh/ruff-pre-commit
   # Ruff version.
-  rev: v0.2.1
+  rev: v0.12.0
   hooks:
     # Run the linter.
     - id: ruff
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)